packages/typespec-python/scripts/regenerate.ts

```diff
@@ -231,7 +231,7 @@ async function _regenerateSingle(spec: string, flags: RegenerateFlags): Promise<
     // Perform some asynchronous operation here
     const options = addOptions(spec, PLUGIN_DIR, flags);
     const commandPromises = options.map((option) => {
-        const command = `tsp compile ${spec} --emit=${PLUGIN_DIR} ${option}`;
+        const command = `tsp compile ${spec} --emit=${toPosix(PLUGIN_DIR)} ${option}`;
```

Comment:
emitter-options shall be posix path style otherwise cause warning: https://github.com/microsoft/typespec/pull/4023